From 532e3f8dc33fb88c3ecc5fa046b2c0e9453dcc21 Mon Sep 17 00:00:00 2001
From: Sandy Huang <hjc@rock-chips.com>
Date: Fri, 7 Sep 2018 11:29:51 +0800
Subject: [PATCH] arm64: dts: rockchip: move reserved-memory to evb dtsi file

The board dts file no need to care about the reserved-memory,
so we move this from rk1808-evb-v10.dts to rk1808-evb.dtsi.

Change-Id: I5ac8ef8e3278491b78d8a46fe839a250017f31dd
Signed-off-by: Sandy Huang <hjc@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk1808-evb-v10.dts | 11 -----------
 arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi    | 11 +++++++++++
 2 files changed, 11 insertions(+), 11 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk1808-evb-v10.dts b/arch/arm64/boot/dts/rockchip/rk1808-evb-v10.dts
index 421ce885b010..d81a5b9b02cb 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808-evb-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/rk1808-evb-v10.dts
@@ -14,15 +14,4 @@
 	chosen {
 		bootargs = "earlycon=uart8250,mmio32,0xff550000 console=ttyFIQ0 root=/dev/mmcblk1p8 rootfstype=ext4 rootwait swiotlb=1 kpti=0";
 	};
-
-	reserved-memory {
-		#address-cells = <2>;
-		#size-cells = <2>;
-		ranges;
-
-		drm_logo: drm-logo@00000000 {
-			compatible = "rockchip,drm-logo";
-			reg = <0x0 0x0 0x0 0x0>;
-		};
-	};
 };
diff --git a/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi b/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
index f1dd3b0988ec..13e5cc5d69d3 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk1808-evb.dtsi
@@ -62,6 +62,17 @@
 		status = "okay";
 	};
 
+	reserved-memory {
+		#address-cells = <2>;
+		#size-cells = <2>;
+		ranges;
+
+		drm_logo: drm-logo@00000000 {
+			compatible = "rockchip,drm-logo";
+			reg = <0x0 0x0 0x0 0x0>;
+		};
+	};
+
 	sdio_pwrseq: sdio-pwrseq {
 		compatible = "mmc-pwrseq-simple";
 		pinctrl-names = "default";
-- 
2.35.3

